### ISSUE TITLE
NESHawk lacks sliders for special sound channels

### ISSUE THREAD

**Issue by despoa** [TASEmulators/BizHawk#3306]
Under Sound Channels is a slider for a setting called "APU Channels". This affects the overall volume of all five standard channels of the NES APU. However, it does not affect the volume of extra channels added by special mappers such as the VRC7, which can be best seen with Lagrange Point. It would be nice if there are extra sliders for these special channels, one for each mapper like with Mesen.

**Comment by Sonia-7** [comment]
I agree with feos. By the way, I compared Akumajou Densetsu (VRC6)'s real hardware audio with the emulator and the special channels also sound too quiet in the latter, which isn't accurate at all.

**Comment by vadosnaprimer** [comment]
> By the way, I compared Akumajou Densetsu (VRC6)'s real hardware audio with the emulator and the special channels also sound too quiet in the latter, which isn't accurate at all.

You have the actual device? Or was it from some video? We searched and found only [these](https://www.youtube.com/playlist?list=PLvqpAsa7-dlwl0bad2nXgsieBI1KVpxti). In the CV3 description the author says it was meant for old famicoms which had quieter external audio and sounds good that way, while Gimmick was meant for newer famicoms like AV which has louder external audio and sounds better _that way_.

**Comment by Sonia-7** [comment]
> You have the actual device? Or was it from some video?

Yes, I do have a Sharp Twin Famicom. Here's how the game's opening sounds like on real hardware:

https://user-images.githubusercontent.com/24638400/178121052-530d0512-9bf5-44ac-b87f-02076d6a1b24.mp4

(Sorry about the bad quality, I have no means of recording the audio internally so just used my phone.)

And here is how it sounds in bizhawk:

https://user-images.githubusercontent.com/24638400/178120913-a40d917f-51f0-4faa-be8b-f0847c04bcd3.mp4

Note how the special channels are too quiet and almost can't be heard over the standard channels. (Setting the APU volume to 10 makes it sound pretty much like the real machine, though.)

**Comment by fsvgm777** [comment]
I...think you got the special channels and the standard channels mixed up. At the default APU volume (1), the standard channels are way too quiet, drowned by the VRC6 audio. However...at APU volume 10, I found that the VRC6 audio gets completely drowned by the standard channels. Setting it to 5 is very close, if not on par to how it sounds like in [this video](https://www.youtube.com/watch?v=Vk-US3QJNck).

**Comment by Sonia-7** [comment]
Yeah, I wasn't completely sure on that regard to be honest. :P
A midpoint between the two sounds like the right thing to do.
